scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -96,6 +96,9 @@ if [ "$os" == "auto" ] ; then
     case $os in
         centos7|centos8|centos9|\
         rhel8|rhel9.1|\
+        opencloudos8.5|opencloudos8.6|opencloudos8.8|opencloudos9.0|opencloudos9.2|\
+        opencloudos2301|opencloudos2305|opencloudos2309|opencloudos2401|opencloudos2403|\
+        tencentos3.1|tencentos3.2|tencentos3.3|tencentos4.0|tencentos4.2|\
```

Comment:
what docker images can I use to test it?

Reply:
opencloudos：
https://hub.docker.com/r/opencloudos/opencloudos/tags

Reply:
tencentos
https://hub.docker.com/r/tencentos/tencentos4-minimal

Reply:
> is it possible to add support to https://github.com/openvinotoolkit/openvino/blob/master/install_build_dependencies.sh as well? I've tried to build inside opencloudos docker image and that script have not installed build dependencies.

ok，I can add it.
But I notice that the conditions in master/install_build_dependencies.sh are different with scripts/install_dependencies/install_openvino_dependencies.sh, for example, there is not anolis or openEuler20.03 branch in install_build_dependencies.sh.
How should add support to install_build_dependencies.sh?

Reply:
`install_build_dependencies.sh` has more generic conditions, which check some files to detect OS.
`install_openvino_dependencies.sh` has conditions on specific OS name and version.

In future we want to unify both scripts to have a common conditions. But currently, you need to have a generic condition like https://github.com/openvinotoolkit/openvino/blob/master/install_build_dependencies.sh#L92 to detect a set of OSes

Reply:
> `install_build_dependencies.sh` has more generic conditions, which check some files to detect OS. `install_openvino_dependencies.sh` has conditions on specific OS name and version.
> 
> In future we want to unify both scripts to have a common conditions. But currently, you need to have a generic condition like https://github.com/openvinotoolkit/openvino/blob/master/install_build_dependencies.sh#L92 to detect a set of OSes

done

Reply:
> I've found that `patchelf` and `ShellCheck` are not available on these docker images out of box. Maybe different names should be used? or some extra repositories should be enabled
> 
> In worst case, we can add conditions to skip these dependencies for these OSes

OK，These packages are supported last month, and synchronizing to the repo may take some time
https://gitee.com/opencloudos-stream/ShellCheck
https://gitee.com/opencloudos-stream/patchelf

I will add conditions temporarily

Reply:
> I've found that `patchelf` and `ShellCheck` are not available on these docker images out of box. Maybe different names should be used? or some extra repositories should be enabled
> 
> In worst case, we can add conditions to skip these dependencies for these OSes

done